src/modules/displayItems.js

```diff
@@ -0,0 +1,35 @@
+import displayPopup from './popupWindow.js';
+import { getLikes, likeInteract } from './like.js';
+import itemsCounter from './itemCounter.js';
+
+const recipeCount = document.querySelector('.item-count');
+const recipeCards = document.querySelector('.recipe-cards');
+
+const displayCards = async () => {
+  fetch('https://www.themealdb.com/api/json/v1/1/filter.php?c=Vegetarian')
+    .then((response) => response.json())
```

Comment:
✅ The web app uses an API selected by students that base its development around it. 👍🏻